tests/test_modeling_tf_bart.py

```diff
@@ -120,37 +118,6 @@ def test_inputs_embeds(self):
         # inputs_embeds not supported
         pass
 
-    def test_compile_tf_model(self):
```

Comment:
Is this test removed because it's in the common tests?

Reply:
Exactly! I think Sam did rewrite it because of the different outputs, but basically there was no diff between the two (this one and the common).